### PR TITLE
Add initial 1.23 folder to solicit feedback on start/end date

### PR DESCRIPTION
**Pull request by jeremyrickard**
As discussed in the SIG Release meeting on Tuesday, this PR creates a scaffold 1.23 directory with a mostly TBD schedule in the README.md file. We are still quite a ways out from 1.23 kicking off, but requested by [this discussion](https://github.com/kubernetes/sig-release/discussions/1566), SIG Release wanted to begin projecting next cycle's release schedule so that the community would have greater awareness and could provide feedback. 

TL;DR:

- **Monday, August 23rd**: Week 1 - Release cycle begins
- **Tuesday, December 7th**: Week 16 - Kubernetes v1.23.0 released

Note: this schedule only includes a proposed start and end date, with KubeCon NA + Colocated events depicted for awareness purposes. The release team will still need to refine the dates for things like enhancement freeze and code freeze, based on input from the community and action items from the 1.22 retrospective.

The proposed 1.23 cycle start and end date are based on the [policy elements](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/2572-release-cadence#schedule-policy) from the release cadence KEP, namely:

* The last Kubernetes release of a year should be finished by the middle of December.
* A Kubernetes release cycle has a length of of ~15 weeks.
* An explicit SIG Release break of at least two weeks between each cycle will be enforced.
* Events like KubeCon will be considered as blocked from development or decision-making from the SIG release perspective and the release team will not hold meetings during this week. The release team must also consider the week before and after the event when setting deadlines to minimize impact on contributors.

Note: This doesn't mean that 1.23 development can't start until August 23rd, thaw of the main branch should still coincide with the 1.22 release. The start date really reflects the kick off of RT ceremonies and activities. 

/kind documentation
/area release-team
/sig release

/hold for discussion and comment 
Signed-off-by: Jeremy <jeremyrrickard@gmail.com>

### CODE REVIEW

File: releases/release-1.23/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+- **Tuesday, December 7th**: Week 16 - Kubernetes v1.23.0 released
+- **TBD**: Week 17 - Release Retrospective
+
+## Timeline
+
+| **What** | **Who** | **When** | **WEEK** | **CI SIGNAL** |
+|---|---|-------|---|---|
+| Start of Release Cycle | Lead | Mon August 23rd | week 1 | [master-blocking] |
+| Start Enhancements Tracking | Enhancements Lead | Mon August 23rd | Week 1 | |
+| 1.230-alpha.1 released | Branch Manager ([TBD](TBD)) | Wed August 25  | Week 1 | |
```
Comment (warmchang) [inline releases/release-1.23/README.md:54]: 1.230 --> 1.23.0

**Review by saschagrunert — APPROVED** [review]

**Comment by tpepper** [comment]
Small request:  Can you add ", 2021" in the date strings?  As the discussions of dates shift to forecasting a year out, having the year will be key to keeping things straight.

File: releases/release-1.23/README.md (hunk 1 above)
Comment (jeremyrickard) [reply releases/release-1.23/README.md:54]: Thanks for catching this

**Comment by saschagrunert** [comment]
@jeremyrickard can we unhold this?

**Comment by jeremyrickard** [comment]
@saschagrunert oh no, I thought I had upheld it a while ago. Yes we should.

/hold cancel

**Pull request merged**